pkg/controller/kabaneroplatform/admission_controller_webhook.go

```diff
@@ -84,7 +85,7 @@ func reconcileAdmissionControllerWebhook(ctx context.Context, k *kabanerov1alpha
 		return err
 	}
 
-	m, err := mf.FromReader(strings.NewReader(s), c)
+	mOrig, err := mf.ManifestFrom(mf.Reader(strings.NewReader(s)), mf.UseClient(mfc.NewClient(c)))
```

Comment:
One other note: you may want to pass `mf.UseLogger(reqLogger.WithName("manifestival"))` or similar to have log what it's doing. Nice choice using `go-logr`, btw!

Reply:
Excellent - we'll do that too.